docs/tutorials/first_steps/05_interactive_exploration.rst

```diff
@@ -10,71 +10,19 @@ our question: *how many flights had Austin, TX as their destination.*
 In this tutorial, we will jump-start the process by using some notebooks that
 have already been prepared.
 
-Use the commands below to add the first notebook to your project.
+Go to `the first example notebook <https://renkulab.io/projects/renku-tutorials/renku-tutorial-flights-material/files/blob/notebooks/FilterFlights.ipynb>`_
+and download the file to your computer. Go to the ``notebooks`` directory in your
```

Comment:
I think these instructions need to be a bit more explicit, maybe containing images.

When I followed the link and clicked on the "Download" icon, I got a file with a `.txt` extension. I had to rename it to a `.ipynb` extension.

Reply:
oh really? that must depend on the browser then (in firefox I get .ipynb)

Reply:
I checked with Chrome and Safari and they both do different things - I added a note about it.